lib/trento/activity_log.ex

```diff
@@ -44,14 +44,24 @@ defmodule Trento.ActivityLog do
 
   @spec list_activity_log() :: list(ActivityLog.t())
   def list_activity_log do
-    # This will be made filterable/paginatable in a later PR
     query =
       from activity in ActivityLog,
         order_by: [desc: activity.inserted_at]
 
     Repo.all(query)
   end
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** Shouldn't this be removed?‏